src/contracts/BPool.sol

```diff
@@ -57,24 +61,40 @@ contract BPool is BToken, BMath, IBPool {
 
   /// @inheritdoc IBPool
   function setSwapFee(uint256 swapFee) external _logs_ _lock_ {
-    require(!_finalized, 'ERR_IS_FINALIZED');
-    require(msg.sender == _controller, 'ERR_NOT_CONTROLLER');
-    require(swapFee >= MIN_FEE, 'ERR_MIN_FEE');
-    require(swapFee <= MAX_FEE, 'ERR_MAX_FEE');
+    if (_finalized) {
+      revert BPool_PoolIsFinalized();
+    }
```

Comment:
can we try to implement 1-liners?
`if (_finalized) revert BPool_PoolIsFinalized();`

whenever possible i'd prefer them, if foundry formats them because of the line length i'm ok

Reply:
oh, i see @0xteddybear's comment, good we can keep it as is 👌